Project.toml

```
@@ -47,7 +47,7 @@ DocStringExtensions = "0.8"
 DynamicPPL = "0.12.1"
 EllipticalSliceSampling = "0.4"
 ForwardDiff = "0.10.3"
-Libtask = "= 0.4.0, = 0.4.1, = 0.4.2, = 0.5.0, = 0.5.1"
+Libtask = "0.4, 0.5"
```

```suggestion
Libtask = "0.4, 0.5.3"
```
Should probably lower-bound to the working version, no?